pkg/api/models/approval.go

```diff
@@ -18,3 +20,21 @@ type Approval struct {
 	// Time of the event
 	Time string `json:"time,omitempty"`
 }
+
+// ToJSON converts object to JSON string
+func (a *Approval) ToJSON() ([]byte, error) {
```

Comment:
let's also add some unit tests for the new functions :)

Reply:
totally agree, I created a separate issue for the unit tests, as it can be quite time-consuming keptn/keptn#6561

Reply:
agreed, fine with me if we do the unit tests in a separate ticket